### ISSUE TITLE
Lost connection to my music folder?

### ISSUE THREAD

**Issue by tudza** [CDrummond/cantata#1558]
I was running Cantata on Kubuntu.  Set it up to point to a local folder in my home directory.  Worked just fine.

I launched it just now and it claims it cannot connect to that folder.  I have no idea what it is expecting.  I'm not streaming music from another location or anything, I just want to play music stored locally and can't figure out what changed.

**Comment by CDrummond** [comment]
What do you mean by "connect to folder"? Cantata connects to MPD. What is the **exact** error message? Do you have MPD running? How is Cantata configured? Basic mode, or standard (server-style) mode?

**Comment by tudza** [comment]
I see what you mean after reading more about Cantata. Being a client of mpd, it makes no sense to talk about the application not talking to a folder.

When I set this up as my music player, all I recall doing was selecting a folder with my music in it under a configuration it has named "Personal" and everything worked.  Hence, a surprise when I launched it and it did not work by showing a red message at the top saying Cantata could not connect to Personal.

So far as I can tell, mpd is not running. That would be the root of the problem.  Why it's not running I can't say.

I'll have to see about getting mpd started, then see what happens in Cantata, and probably have to fix the mess I've made of the configuration by trying to fix the problem changing configurations.
